birzebbuga.md

```diff
@@ -0,0 +1,5 @@
+## Birzebbuga, Malta
```

Comment:
@KLia Would love to visit Malta someday! When is a good time of the year to visit?